_monitoring-your-cluster/job-scheduler/locks.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: default
+title: Locks API
+parent: Job Scheduler
+nav_order: 20
+redirect_from:
+    - /monitoring-plugins/job-scheduler/api/
+---
+
+# Job Scheduler Locks API 
+Introduced 3.2
+{: .label .label-purple }
+
+The Job Scheduler uses a distributed locking mechanism to ensure that only one instance of a job runs at a time across the cluster. The Locks API returns information about all active job locks managed by the Job Scheduler.
+
+## Endpoints
+
+```json
+GET /_plugins/_job_scheduler/api/locks
+GET /_plugins/_job_scheduler/api/locks/<lock_id>
```

Comment:
@Jeremydupras Please check the endpoint on line 20. I assume it has to have "locks" in the path, so I changed it.